storehaus-memcache/src/main/scala/com/twitter/storehaus/memcache/MemcacheLongStore.scala

```diff
@@ -34,9 +35,16 @@ import com.twitter.bijection.netty.ChannelBufferBijection
  */
 object MemcacheLongStore {
   private implicit val cb2ary = ChannelBufferBijection
+  private implicit val b2ary = new Bijection[Buf, Array[Byte]] {
+    def apply(buf: Buf) = Buf.ByteArray.Owned.extract(buf)
```

Comment:
This is more of a variation in style I guess. Personally, override is clearer to the reader without having to look if it comes from the base trait. Maybe doesn't make sense to do this for `apply`.